generated/nirfmxlte/nirfmxlte.proto

```diff
@@ -2566,7 +2581,10 @@ message CfgPHICHResponse {
 message CfgPSSCHModulationTypeRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
-  int32 modulation_type = 3;
+  oneof modulation_type_enum {
```

Comment:
This is a binary breaking change as is, consider adding something like
```python
'grpc_field_number': '4',
'grpc_raw_field_number': '3',
```
to the RFmxLTE `functions.py` file for the modulationType param. That'll at least make it so old clients will still work passing along the int value for field 3.